### DISCUSSION TITLE
Update 5.10.160 to 6.1

### DISCUSSION THREAD

**Discussion by nguyenhunga5** [Joshua-Riek/ubuntu-rockchip#725] [General]
Greetings,

I hope this message finds you well. I'm interested in updating my Orange PI 5 from the current kernel version 5.10.160 to version 6.1. Given that I have numerous applications installed, I was wondering if you could guide me through the process of upgrading to 6.1. Any assistance you can provide would be greatly appreciated.

Thank you in advance for your help.

Best regards,

**Comment by Xeue** [comment]
There's been a few posts about this so far, a lot has changed under the hood in the kernel, so this isn't really possible in any way that is remotely straight forwards...

**Reply by Joshua-Riek** [reply]
Correct. I do not have the time to work on an upgrade path.

**Reply by nguyenhunga5** [reply]
thank you so much for your work.
